datasets/noaa-mrms-qpe/tests/test_dataset.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from pathlib import Path
+
+from pctasks.cli.cli import setup_logging, setup_logging_for_module
+from pctasks.dev.test_utils import run_process_items_workflow
+
+HERE = Path(__file__).parent
+DATASET_PATH = HERE / "dataset-dev-cluster.yaml"
+
+
+def test_1h_pass1():
+    run_process_items_workflow(
+        DATASET_PATH,
+        "noaa-mrms-qpe-1h-pass1",
+        args={
+            "registry": "localhost:5001",
+        },
+        splits_limit=1,
+        chunks_limit=2,
+        timeout_seconds=600,
+    )
+
+
+def test_1h_pass2():
+    run_process_items_workflow(
+        DATASET_PATH,
+        "noaa-mrms-qpe-1h-pass2",
+        args={
+            "registry": "localhost:5001",
+        },
+        splits_limit=1,
+        chunks_limit=2,
+        timeout_seconds=600,
+    )
+
+
+def test_24h_2pass():
+    run_process_items_workflow(
+        DATASET_PATH,
+        "noaa-mrms-qpe-24h-pass2",
+        args={
+            "registry": "localhost:5001",
+        },
+        splits_limit=1,
+        chunks_limit=2,
+        timeout_seconds=600,
+    )
+
+
+if __name__ == "__main__":
```

Comment:
Is this needed? We could / should be using pytest or unittest's test discovery to run these.

Reply:
That's a pattern I put into the dataset integration tests; during debugging a dataset integration tests it was more useful to have it run directly instead of through pytest (I forget exactly why, perhaps because pytest doesn't show output during the test run but only after failure). I'm fine with keeping or leaving out if deduplication needs pytest as mentioned above.

Reply:
Yep, it is nice to see the output in real time. But for now I've switched to using pytest and can eliminate this.

You can append `-s` to the pytest command, but then you end up with pytest results mixed in with the output.